src/tests/calculator.test.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Calculator from '../pages/Calculator';
+
+it('renders correctly', () => {
+  const tree = renderer
+    .create(<Calculator />)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Please fix the error.

![image](https://user-images.githubusercontent.com/38635158/160005441-f1bd4405-dce5-4de0-a7b2-7f031f05d77d.png)